jazz_create-serverless-service/index.js

```diff
@@ -31,15 +31,17 @@ const crud = require("./components/crud")(); //Import the utils module.
     @author:
     @version: 1.0
 **/
-
-module.exports.handler = (event, context, cb) => {
+var user_id =  undefined;
+var serviceId;
+var serviceDataObject;
+var handler = (event, context, cb) => {
 
     var errorHandler = errorHandlerModule();
     var config = configObj(event);
     logger.init(event, context);
 
-    var serviceId;
-    var serviceDataObject;
+    serviceId;
```

Comment:
Will Remove the 2 statements : 
Putting the variables as global because variables are used in the subsequent functions that are called in index. handler, due to which the lifetime of the variable is the entire lifetime of index handler - ie the entire lifetime of this API execution. Declaring them global as to avoid increasing the parameters passed of the subfunctions and keeping them as modular as possible